### ISSUE TITLE
Load the hadith from local.storage

### ISSUE THREAD

**Issue by TheAdnan** [TheAdnan/hadith-of-the-day#2]
Note: Include `"storage"` in `manifest.json `

**Comment by TheAdnan** [comment]
For reference, take a look at the storage handling methods in this script:
[github.com/TheAdnan/Session-resurrection/blob/master/sessions-pop-up/save-session.js](https://github.com/TheAdnan/Session-resurrection/blob/master/sessions-pop-up/save-session.js)

**Comment by Abdur-rahmaanJ** [comment]
received

**Comment by Abdur-rahmaanJ** [comment]
currently it loads from a bukhari.json. where should i add a menu to let users load their own hadith.json? by rightclick and selecting menu or the issue wants something else?

**Comment by TheAdnan** [comment]
That won't be necessary, just make sure that the hadith load from the local.storage, and if the storage is empty, set it using the data from the bukhari.json file.

**Comment by Abdur-rahmaanJ** [comment]
you've already done it here 
https://github.com/TheAdnan/hadith-of-the-day/blob/master/extension/hadith/script.js

first you checked for null in local storage the if it is you set local storage to bukari.json

**Comment by TheAdnan** [comment]
Nope, it doesn't, I've tried but I didn't know how to set it at the time.

**Comment by JafarAkhondali** [comment]
Where is bukari.json ?!

**Comment by hkicin1** [comment]
:cat:
